core/property/registry.go

```diff
@@ -2,31 +2,68 @@ package property
 
 import (
 	"fmt"
-	"sync"
-
+	"github.com/TIBCOSoftware/flogo-lib/config"
 	"github.com/TIBCOSoftware/flogo-lib/core/data"
 	"github.com/TIBCOSoftware/flogo-lib/logger"
+	"reflect"
+	"regexp"
+	"sync"
 )
 
 var (
-	props = make(map[string]string)
-	mut   = sync.RWMutex{}
+	props    = make(map[string]interface{})
+	mut      = sync.RWMutex{}
+	regex    = regexp.MustCompile(config.GetPropertyDelimiterFormat())
+	resolver Resolver
 )
 
+func init() {
```

Comment:
It should not. remove init funcion in property/registry.go and put it into environment.go.  there is no data package import in property folder so it should fine.

BTW, without this fix. there has issue while user contributes resolver. Customer resolver will not take effect while this go file initializes later than custom resolver.